### PR TITLE
Us33953 new

### PR DESCRIPTION
**Pull request by mandeep-singh-lab49**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: e2e-test.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+npx hardhat run ./deployment/scripts/transparentUpgradeableProxy.ts;
+
+sed -i~ '/^CONTRACT_NAME=/s/=.*/="LPToken"/' .env
+
+npx hardhat run ./deployment/scripts/logic.ts;
+npx hardhat run ./deployment/scripts/transparentUpgradeableProxy.ts;
+
+npm run e2e-test
+
+echo "********************  Done *************************************";
```
Comment (abhishek-k-sharma-lab49) [inline e2e-test.sh:46]: update this

File: .github/workflows/e2e-test.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest 
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+      - run: npm ci
+      - name: Running build and deployment
```
Comment (abhishek-k-sharma-lab49) [inline .github/workflows/e2e-test.yml:36]: new comment

File: e2e-test/features/PairTest.feature [hunk 3]
```diff
@@ -10,12 +10,12 @@ Feature: Pair contract e2e test
...
+        Then User verifies 205 units of tokenA and 225 units of tokenB are left in pool
     
     Scenario: Verify swapping tokenA increase the tokenA quantity and decreases tokenB quantity
         Given tokenA and tokenB are present in pool
         When User swap 1 unit of tokenA
-        Then increased tokenA quantity is 206 and decreased tokenB quantity is 223 in pool
+        Then increased tokenA quantity is 206 and decreased tokenB quantity is 224 in pool
 
     Scenario: Verify sport price for tokenA 
         Given tokenA and tokenB are present in pool
```
Comment (abhishek-k-sharma-lab49) [inline e2e-test/features/PairTest.feature:21]: asdads

**Review by abhishek-k-sharma-lab49 — COMMENTED** [review]
adasd

**Review by abhishek-k-sharma-lab49 — APPROVED** [review]

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Pull request merged**